forest/daemon/src/main.rs

```diff
@@ -125,6 +126,10 @@ fn check_for_low_fd(config: &Config) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+#[cfg(feature = "paritydb")]
+fn check_for_low_fd(_config: &Config) -> Result<(), anyhow::Error> {
```

Comment:
So no more checks are needed with ParityDb?

Reply:
I was not able to get to a point where FD limit struck Forest so I assume it is fine for now.